Test/collectiontest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+require_once('../functions.php');
+
+use PHPUnit\Framework\TestCase;
+
+class collectiontest extends TestCase {
+    public function testReturnsCharacterData() {
+        $expected = "<div class=\"character test house\"><h3>Test Name</h3><p>Test Blood-Type</p><p>Test House</p></div>";
```

Comment:
if you use single quotes to contain the string you wont have to escape the double quotes